src/screens/TripDetailsScreen.tsx

```
@@ -91,16 +143,83 @@
+
+      {isPaymentModalOpen && (
+        <div className="modal is-active">
+          <div className="modal-background" onClick={() => setIsPaymentModalOpen(false)}></div>
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <p className="modal-card-title">{paymentToEdit ? 'Edit Payment' : 'New Payment'}</p>
+              <button
+                className="delete"
+                aria-label="close"
+                onClick={() => setIsPaymentModalOpen(false)}
+              ></button>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)